JS/cart.js

```diff
@@ -0,0 +1,49 @@
+// Render Cart
```

Comment:
Redundant comment. Don't write a comment for obvious stuff, a comment should be a description for a complex logic that would would take time to read the code to understand it or explain why you have done something in a certain way.